src/mon/Monitor.cc

```diff
@@ -5458,7 +5453,7 @@ vector<DaemonHealthMetric> Monitor::get_health_metrics()
   utime_t oldest_secs;
   const utime_t now = ceph_clock_now();
   auto too_old = now;
-  too_old -= g_conf->get_val<double>("mon_op_complaint_time");
+  too_old -= g_conf->get_val<std::chrono::seconds>("mon_op_complaint_time");
```

Comment:
we don't have `operator-=(utime& t, std::chrono::seconds)` yet, probably could just put
```c++
too_old -= g_conf->get_val<std::chrono::seconds>("mon_op_complaint_time").count();
```